ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Issue and Steps to Reproduce
+<!-- Describe your issue and tell us how to reproduce it (include any useful information). -->
+
+### Versions
```

Comment:
Is this asking for Git/LFS versions? Maybe some debug instructions could be added to the main issue description? 

* Run `git lfs env` in your repository working directory to display the current version of Git and LFS, and current git config.
* Run command with `GIT_TRACE=1` to get debug messages in the output.

Also, it should probably have some sample command output in the markdown, because people frequently forget to add the ``` fencing blocks.

```
    ```sh
    $ git push
    PROBLEM OUTPUT
    PROBLEM OUTPUT
    PROBLEM OUTPUT
    ```
```